packages/console/src/components/AppContent/components/Sidebar/hook.tsx

```diff
@@ -48,7 +47,9 @@ export const useSidebarMenuItems = (): {
   const {
     data: { getStartedHidden },
   } = useUserPreferences();
-  const language = useLanguage();
+  const {
+    i18n: { language },
+  } = useTranslation();
```

Comment:
so `useLanguage` is not needed anymore?

Reply:
You know what... I think these late night code need to be refactored. I take a second look at these changes and now have the same question for myself. 😂

Reply:
lol